docs-client/src/components/VariableList/index.tsx

```diff
@@ -237,7 +243,6 @@ export default ({ title, variables, specification }: Props) => {
           </TableBody>
         </Table>
       </TableContainer>
-      <Typography variant="body2" paragraph />
```

Comment:
This might have been intentional, perhaps to make sure there's more margin at the bottom of each variable list. If so, I'll revert this.

Reply:
Looks great to me. Thanks, @KarboniteKream :bow: